source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,597 @@
+
+The ovirt-provider-ovn translates the security group information into a
+port group object, encoding the security group information in the
+external ids column.
+
+Create/update timestamps of the security groups are also implemented,
+thus enabling the [Resource timestamps](https://developer.openstack.org/api-ref/network/v2/#id367)
+networking-api extension.
+
+The security group also has a *revision_number* attribute - also stored
+in the external ids - that is bumped everytime a change is made to the
+security group - add / remove rules, update name or description.
+
+A sequence diagram depicting the creation of a security group meant for
+holding rules to allow incoming ssh traffic is shown below.
+
+![add-sec-group](../../../../images/features/network/security-groups/create_security_group_ansible.png)
+
+### Add security group rule
+
+Adding security group rules is also done out of band, using the [os_security_group_rules](https://docs.ansible.com/ansible/2.5/modules/os_security_group_rule_module.html) or sending its correspondent REST API request.
+
+The ACL match condition is built from the *ether_type*, *protocol*,
+*port_range_max*, *port_range_min*, and *remote_ip_prefix* security
+group rules attributes.
+
+All of those attributes are then stored in the ACL external_ids column -
+along with the *description* and *security group id* attributes.
+
+The newly created ACL is then stored in the ovn-nb database, and its
+data is afterwards translated back into the networking-api syntax.
+
+The security group rules also feature the resource timestamps extension,
+but the revision number will always be *one*, since we do not allow a
+rule to be updated.
+
+A sequence diagram where a user adds - through ansible - a security
+group rule meant to white list ssh traffic is added to the security
+group 'allow-ssh' is shown below.
+
+![add-rule](../../../../images/features/network/security-groups/create_security_group_rule_ansible.png)
+
+### Creating a subnet on top of a network
+
+Creating a Networking API subnet leads to the creation of its
+corresponding DHCP options object in the OVN north database.
+
+If the feature is activated - e.g. the **deny-all** port group is
+present in the OVN northbound database - the DHCP related traffic will
+be dropped by Open vSwitch. To prevent that, thus assuring that the VMs
+can get an IP address through DHCP, a security group meant to hold the
+rules white listing DHCP traffic is created, and ACLs allowing that
+traffic added to it.
+
+The following sequence diagram depicts the flow described above.
+
+![add-subnet](../../../../images/features/network/security-groups/create_subnet.png)
+
+### Attach a VM to an external network.
+
+Please refer to the sequence diagram below to better understand this
+flow.
+
+When a VM is attached to an external network, an ovn logical switch port
+is created. This port concept is already implemented in the
+ovirt-provider-ovn project, *but* does not allow it to be bound to
```

Comment:
I would emphasize the "not", but it's not a must

Reply:
I prefer it as is.